tasks/libs/common/worktree.py

```diff
@@ -51,8 +51,16 @@ def init_env(ctx, branch: str | None = None):
             ctx.run(f"git -C '{WORKTREE_DIRECTORY}' checkout '{branch}'", hide=True)
 
         if not os.environ.get("AGENT_WORKTREE_NO_PULL"):
+            # TODO A
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is this `TODO` a leftover?

Reply:
Oops, fixed ! Thanks